r_scripts/load_data.r

```diff
@@ -123,6 +123,36 @@ load_field_grav <- function(){
                   )
 
 }
+
```

Comment:
It looks like there is two copies of load_field_grav in this file, is it okay to delete one?